docs/policies.rst

```diff
@@ -393,5 +394,78 @@ It is recommended to use
           ``mu_neg = mu_pos`` and ``use_max_sim_neg = False``. See
           `starspace paper <https://arxiv.org/abs/1709.03856>`_ for details.
 
+Two Stage Fallback Policy
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This policy handles low NLU confidence in multiple stages.
+
+- If a NLU prediction has a low confidence score, the user is asked to confirm
+  whether they really had this intent.
+
+    - If they confirm, the story continues as if the intent was recognized
+      with high confidence from the beginning.
+    - If they deny, the user is asked to restate his intent.
```

Comment:
please don't use gender specific pronouns (always their)

Reply:
also if you're calling it clarification down there, i'd say "the user is asked to clarify their message" to be consistent

Reply:
👍 but I'd keep intent in there - `the user is asked to clarify their intent`

Reply:
yeah i just meant consistency between the two :D